tests/test_config.py

```diff
@@ -148,27 +147,26 @@ def test_configfile_hierarchy(monkeypatch, tmpdir):
             """
 [database]
 host=host3
+username=non-default-name-1
 [influxdb]
 tags=tag2=value2
-[dashboard]
-path=/some/other/directory
         """
         )
 
     with open(dot_inmanta_cfg_file, "w", encoding="utf-8") as f:
         f.write(
             """
-[dashboard]
-path=/directory
-client-id=test123
+[database]
```

Comment:
This test case is updated to remove the `dashboard.path` and the `dashboard.client-id` config options. That's good. But these options are not 1-to-1 replaced with another option. That way the test doesn't test what is used to test. Why was this done?

Reply:
Ah indeed, the mapping was wrong